src/Executables/Game/SkillManager.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Immutable;
+using Microsoft.Extensions.Logging;
+using QuantumCore.API;
+using QuantumCore.API.Core.Models;
+
+namespace QuantumCore.Game;
+
+/// <summary>
+/// Manage all static data related to skills
+/// </summary>
+public class SkillManager : ISkillManager
+{
+    private readonly ILogger<SkillManager> _logger;
+    private ImmutableArray<SkillData> _skills = ImmutableArray<SkillData>.Empty;
+    
+    public SkillManager(ILogger<SkillManager> logger)
+    {
+        _logger = logger;
+    }
+    
+    public SkillData? GetSkill(uint id)
+    {
+        return _skills.FirstOrDefault(skill => skill.Id == id);
+    }
+
+    public SkillData? GetSkillByName(ReadOnlySpan<char> name)
+    {
+        foreach (var dataSkill in _skills)
+        {
+            if (name.Equals(dataSkill.Name, StringComparison.InvariantCulture))
+            {
+                return dataSkill;
+            }
+        }
+        return null;
+    }
+
+    public async Task LoadAsync(CancellationToken token = default)
+    {
+        if (_skills.Length > 0)
+        {
+            return;
+        }
+
+        _skills = await ParserUtils.GetSkillsAsync("data/936skilltable.txt", token);
```

Comment:
Why this random name?

Reply:
It's actually not random, it's the filename that client applications use to also have the skills information. We can rename them to whatever we want in the server. Maybe something like `skilltable.txt` ?

Reply:
On a second check, it's actually `skilltable.txt`, not sure where I got the `936` from.